spec/02-integration/09-hybrid_mode/11-status_spec.lua

```diff
@@ -106,64 +103,59 @@ for _, strategy in helpers.each_strategy() do
         end, 10)
       end)
 
-      -- now dp receive config from cp, so dp should be ready
-
-      skip_rpc_sync("should return 200 on data plane after configuring", function()
```

Comment:
I think that you misunderstand my words, we still need this case for rpc_sync = off but not use `skip_rpc_sync`, it should be a normal `it()`

Reply:
fixed

Reply:
It should also support `{"off", "off"}`

Reply:
note that "off" "off" is supported by original test case,
I think we should write a complete on on case, like

```
if off off
-- using original cases
end

if on on then
-- write new cases in this pr
describe ()
....
end
end
```

So the new case does not need to support on

Reply:
because the original case (off off) will restart CP, if we implement new case in this scenario, current CP/DP incremental sync is hard to make the case stable, it is easy to get flakiness.

Why?

because restart CP will cause DP connecting CP and introduce some delay, and current in incremental sync feature, we could not have a way to make sure we have established CP and DP. I think if we want to test CP/DP restart , we could file a new kag to track.